src/components/structures/TagPanel.js

```diff
@@ -60,7 +61,9 @@ const TagPanel = React.createClass({
 
     componentWillUnmount() {
         this.unmounted = true;
+        if (!this.context.matrixClient) return;
         this.context.matrixClient.removeListener("Group.myMembership", this._onGroupMyMembership);
+        this.context.matrixClient.removeListener("sync", this._onClientSync);
```

Comment:
This is a typo, right? Should be `this.onClientSync` I think

Reply:
Yep, thanks @krombel